tinycurl.h

```diff
@@ -58,12 +58,13 @@ class TinyCurl
     TinyCurl(const TinyCurl &);
 
     void setOptions();
-    void setHTTPHeaders(const headers_t &hdrs) const;
+    void setHTTPHeaders(const headers_t &hdrs);
 
     void curlPerform();
 
     CURL *m_handle;
     std::string m_url;
     std::string m_data;
     std::string m_upload_data;
+    struct curl_slist *opt_list = NULL;
```

Comment:
I'm not sure this will work with C++ 98

Reply:
I have no C++98 compiler at hand, but since this is libcurl interface,which is plain old C, maybe its worth a try...